src/python/m5/ext/pystats/__init__.py

```diff
@@ -24,22 +24,11 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from .abstract_stat import AbstractStat
-from .group import Group
-from .jsonloader import JsonLoader
-from .serializable_stat import SerializableStat
-from .simstat import SimStat
-from .statistic import Statistic
-from .storagetype import StorageType
-from .timeconversion import TimeConversion
-
-__all__ = [
-    "AbstractStat",
-    "Group",
-    "SimStat",
-    "Statistic",
-    "TimeConversion",
-    "StorageType",
-    "SerializableStat",
-    "JsonLoader",
-]
+from .abstract_stat import *
+from .group import *
+from .jsonloader import *
+from .serializable_stat import *
+from .simstat import *
+from .statistic import *
+from .storagetype import *
+from .timeconversion import *
```

Comment:
I really don't like this. From [PEP008](https://peps.python.org/pep-0008/#imports)

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools.